src/chrome/content/rules/blob.core.windows.net.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+	For other Microsoft coverage, see Microsoft.xml.
```

Comment:
If you rename the file you also have to update the reference in `Microsoft.xml`.